astropy/stats/tests/test_funcs.py

```diff
@@ -242,6 +242,11 @@ def test_biweight_midvariance_axis_3d():
         bwi = np.array(bwi)
         assert_allclose(bw[y], bwi)
 
+def test_biweight_midcovariance():
+    """test biweight_midcovariance method"""
+    d = np.array([np.random.normal(0,1,500), np.random.normal(0,5,500)])
+    cov = funcs.biweight_midcovariance(d)
+    assert_allclose(cov.shape[0], d.shape[0])
```

Comment:
You don't want only to make sure that the arrays shape match, but also that your implementation of ``biweight_midcovariance`` gives appropriate results.